src/plugins/data_source/server/index.ts

```diff
@@ -8,7 +8,7 @@ import { PluginConfigDescriptor, PluginInitializerContext } from 'src/core/serve
 import { DataSourcePlugin } from './plugin';
 
 export const configSchema = schema.object({
-  enabled: schema.boolean({ defaultValue: false }),
+  enabled: schema.boolean({ defaultValue: true }),
```

Comment:
use false as default. Finally it will be aligned to a switch in global settings

Reply:
done